go.mod

```diff
@@ -33,7 +33,7 @@ require (
 	github.com/ipfs/go-ipld-cbor v0.0.6
 	github.com/ipfs/go-log/v2 v2.5.1
 	github.com/ipfs/go-metrics-prometheus v0.0.2
-	github.com/ipld/go-car v0.3.3
+	github.com/ipld/go-car v2.3.0
```

Comment:
Since this is v2 the path must be: `github.com/ipld/go-car/v2`

Reply:
@bhaskarvilles please tag me for a second review once this has been addressed. Currently, it is still unresolved.

Reply:
@frrist Resolved, now you can merge changes.